Formula/databend.rb

```diff
@@ -0,0 +1,28 @@
+class Databend < Formula
+  desc "Elastic, reliable serverless data warehouse, offers blazing fast query"
+  homepage "https://databend.rs/"
+  url "https://github.com/datafuselabs/databend/archive/refs/tags/v0.6.36-nightly.tar.gz"
+  sha256 "73c4957bd51745b4bacd16f6b7324e14f74b68b1ff53b165ad780e8b23667876"
+  license "Apache-2.0"
+
+  livecheck do
+    url :url
+  end
```

Comment:
Why is this needed?

Reply:
to auto update

Reply:
I think you misunderstood me: my point was the the livecheck should already work even if you delete these three lines.

Reply:
Yes, livecheck checks `url` by default, so there is no need to add this.